node/src/BaseClient.ts

```diff
@@ -1309,6 +1310,24 @@ export class BaseClient {
         return this.createWritePromise(createPersist(key));
     }
 
+    /**
+     * Blocking list pop primitive.
+     * Pop an element from the tail of the first list that is non-empty,
+     * with the given keys being checked in the order that they are given.
+     * Blocks the connection when there are no elements to pop from any of the given lists.
+     * See https://redis.io/commands/brpop/ for more details.
+     *
+     * @param keys - The `keys` of the lists to pop from.
+     * @param timeout - The `timeout` in seconds.
+     * @returns - `Array` of the `key` that elements were poped from, and the popped elements, or null if timeout is reached.
```

Comment:
The return value isn't clear and accurate. it returns only a single element, and a single key, right?
suggestion:
@return - An `array` containing the `key` from which the element was popped and the value of the popped element, formatted as [key, value]. If no element could be popped and the timeout expired, returns Null.

Reply:
Please also try to add examples as we do in python